.github/workflows/test-load-k6.yaml

```diff
@@ -0,0 +1,64 @@
+name: k6 Load Test
+
+on:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+
+jobs:
+  run-tests:
+    name: Run k6 tests
+    runs-on: ubuntu-24.04
+    services:
+      # Datadog agent for collecting OTEL metrics from k6
+      datadog:
+        image: datadog/agent:7-full
```

Comment:
❓ Should / can this use a hash and version tag? I would hope we can pin like we do other things. Also, can Renovate maintain this?

Reply:
I can supply a SHA256 hash if you would like? I am not sure on Renovate maintaining it, but I would assume so if the docker config is being used.